src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http3Connection.cs

```diff
@@ -481,6 +481,10 @@ private async Task AcceptStreamsAsync()
                     _ = ProcessServerStreamAsync(stream);
                 }
             }
+            catch (QuicOperationAbortedException)
+            {
+                // Shutdown initiated by us, no need to abort.
```

Comment:
Would it be better to make a second Abort a no-op? I see the same pattern down below in `ProcessServerStreamAsync` -- is possibly exists in other places as well: 
```c#
try
{ ... }
catch (Exception ex)
{
    Abort(ex);
}
```
so it may be better to treat it inside `Abort`?

Reply:
This doesn't come from an abort though, this is a reaction to a normal shutdown initiated by us. The `QuicOperationAbortedException` here is thrown by the connection as reaction to `AcceptQueue` being closed:
https://github.com/dotnet/runtime/blob/65bcecebde4aebd5926952ccfc4dc8e7fbf7eaed/src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicConnection.cs#L277-L288

The code you're referencing is handling `QuicOperationAbortedException` caused by a different action:
https://github.com/dotnet/runtime/blob/65bcecebde4aebd5926952ccfc4dc8e7fbf7eaed/src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicStream.cs#L270-L276

So, I don't think combining them is a correct solution here. 

I do concur that the utilization of `QuicOperationAbortedException` to notify the user that the connection won't provide any more stream from `AcceptStreamAsync` might be questionable here. But that's not a goal of this PR, I'm just trying to make it work.